### DISCUSSION TITLE
Parcours vs Trace vs Tour

### DISCUSSION THREAD

**Discussion by FJBDev** [mytourbook/mytourbook#1179] [Polls]
Pour les utilisateurs de MT en francais, je me demandais recemment si le mot actuel utilise dans MT "Parcours" est le mieux pour traduire le mot original en anglais "Tour".
J'ai une preference pour le mot "Trace" mais votre avis est le bienvenu

Merci!

**Comment by GurVJ** [comment]
Pour moi aucun ne convient vraiment car certains enregistrements du "Livre des Parcours" n'ont en fait pas de traces GPS, c'est le cas notamment des sessions de Home Trainer, séances de muscu ou renfo., etc.
Je proposerais plutôt quelque chose de plus générique comme "Activité" (éventuellement "Séance", "Session",...)

**Reply by FJBDev** [reply]
J'ai ajoute le mot "Activité" au vote
